site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -157,3 +151,27 @@ automatically for you:
             ...
         }
     }
+
+.. _nested-log:
+
+Nested log
+----------
+
+You can set the logs from other RequestLog_:
+
+.. code-block:: java
+
+    import com.linecorp.armeria.common.logging.RequestLogBuilder;
+
+    RequestLogBuilder.addChild(RequestLog);
+
+If the added RequestLog_ is the first child, the request-side logs of the RequestLog_ will be propagated to the
+parent log. You can add as many children logs as you want, but the rest of logs would not effect.
+If you want to fill the response-side logs of the parent log, please invoke:
+
+.. code-block:: java
+
+    RequestLogBuilder.endResponseWithLastChild();
+
+This will propagate the response-side logs of the last added child. :ref:`retry-with-logging` is logging
```

Comment:
- logs -> log
- child -> child to the parent log
- ... is logging with this feature -> RetryingClient is a good example that leverages this feature. See ... for more information.